custom_model_runner/datarobot_drum/drum/adapters/r/fit.R

```diff
@@ -134,16 +134,17 @@ process_weights <- function(X, weights_filename, weights, na_rows, sample_rows){
                   "is not one of the columns in your training data", sep=' '))
     }
     row_weights <- X[, weights, drop=FALSE]
+    X <- X[,!(colnames(X) %in% c(weights)), drop=FALSE]
```

Comment:
We never separated row_weights from X in R, if X contained the row_weights column :grimacing: